pkg/utils/utils.go

```diff
@@ -70,6 +86,26 @@ func GetValidCniPath(inGoPath string) (string, error) {
 	return cniPath, nil
 }
 
+func GetValidKubeConfig() string {
+	kcPath := filepath.Join(pwd, kcRelPath)
```

Comment:
@iaguis Updated. Now it uses `os.Getwd()` to get the current working directory, and prints out the whole abspath.